services/pub/pub-popularity.tester.js

```diff
@@ -0,0 +1,26 @@
+import Joi from 'joi'
+import { createServiceTester } from '../tester.js'
+
+export const t = await createServiceTester()
+
+t.create('pub popularity (valid)')
+  .get('/analysis_options.json')
+  .expectBadge({
+    label: 'popularity',
+    message: Joi.string().regex(/^\d+%$/),
```

Comment:
We have an existing test validator for this: isIntegerPercentage
https://github.com/badges/shields/blob/f219e3cd193426807a5145d4992ebccebc40ec6a/services/test-validators.js#L94

Reply:
I have added the isIntegerPercentage for popularity scores.